src/runners/http_rfqm_service_runner.ts

```diff
@@ -0,0 +1,125 @@
+ */
+import { createDefaultServer } from '@0x/api-utils';
+import { ProtocolFeeUtils, QuoteRequestor } from '@0x/asset-swapper';
+import { NULL_ADDRESS } from '@0x/utils';
+import Axios, { AxiosRequestConfig } from 'axios';
+import * as express from 'express';
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
+import { Agent as HttpAgent, Server } from 'http';
+import { Agent as HttpsAgent } from 'https';
+
+import { getContractAddressesForNetworkOrThrowAsync } from '../app';
+import {
+    CHAIN_ID,
+    defaultHttpServiceWithRateLimiterConfig,
+    ETH_GAS_STATION_API_URL,
+    META_TX_WORKER_REGISTRY,
+    RFQM_MAKER_ASSET_OFFERINGS,
+    RFQT_MAKER_ASSET_OFFERINGS,
+    RFQ_PROXY_ADDRESS,
+    RFQ_PROXY_PORT,
+    SWAP_QUOTER_OPTS,
+} from '../config';
+import { KEEP_ALIVE_TTL, PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS, RFQM_PATH } from '../constants';
+import { logger } from '../logger';
+import { addressNormalizer } from '../middleware/address_normalizer';
+import { errorHandler } from '../middleware/error_handling';
+import { createRfqmRouter } from '../routers/rfqm_router';
+import { RfqmService } from '../services/rfqm_service';
+import { HttpServiceConfig } from '../types';
+import { ConfigManager } from '../utils/config_manager';
+import { providerUtils } from '../utils/provider_utils';
+
+process.on('uncaughtException', (err) => {
+    logger.error(err);
+    process.exit(1);
+});
+
+process.on('unhandledRejection', (err) => {
+    if (err) {
+        logger.error(err);
+    }
+});
+
+if (require.main === module) {
+    (async () => {
+        const provider = providerUtils.createWeb3Provider(defaultHttpServiceWithRateLimiterConfig.ethereumRpcUrl);
+        const config: HttpServiceConfig = {
+            ...defaultHttpServiceWithRateLimiterConfig,
+            // Mesh is not required for Rfqm Service
+            meshWebsocketUri: undefined,
+            meshHttpUri: undefined,
+        };
+
+        const contractAddresses = await getContractAddressesForNetworkOrThrowAsync(provider, CHAIN_ID);
+        const quoteRequestor = new QuoteRequestor(
+            RFQT_MAKER_ASSET_OFFERINGS,
+            RFQM_MAKER_ASSET_OFFERINGS,
+            Axios.create(getAxiosRequestConfig()),
+            undefined, // No Alt RFQM offerings at the moment
+            logger.warn.bind(logger),
+            logger.info.bind(logger),
+            SWAP_QUOTER_OPTS.expiryBufferMs,
+        );
+
+        const protocolFeeUtils = ProtocolFeeUtils.getInstance(
+            PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS,
+            ETH_GAS_STATION_API_URL,
+        );
+        const metaTxWorkerRegistry = META_TX_WORKER_REGISTRY || NULL_ADDRESS;
+        const rfqmService = new RfqmService(quoteRequestor, protocolFeeUtils, contractAddresses, metaTxWorkerRegistry);
+
+        const configManager = new ConfigManager();
+
+        await runHttpRfqmServiceAsync(rfqmService, configManager, config);
```

Comment:
Just checking, can you confirm that Prometheus is working correctly? are you able to read the metrics exposed by Prometheus? I noticed some runners need to explicitly enable Prometheus, do we need to do the same here?

Reply:
verified that `createDefaultServer` will do this automatically: https://github.com/0xProject/api-utils/blob/main/src/default_server.ts#L72